backend/operations_scanner.go

```diff
@@ -123,35 +129,32 @@ func (s *OperationsScanner) pollDBOperations(ctx context.Context, logger *slog.L
 	}
 }
 
-func (s *OperationsScanner) pollCSOperations(ctx context.Context, logger *slog.Logger, stop <-chan struct{}) {
+func (s *OperationsScanner) pollCSOperations(logger *slog.Logger) {
 	var activeOperations []*database.OperationDocument
 
+	ctx := context.Background()
```

Comment:
nit: let's add a note to why this context is needed (we are performing write operations that should not be canceled)

Reply:
Will do.  Worth mentioning, too, is once we start using Cosmos DB transactions (soon) this concern goes away.